app/src/Lib/ConsoleOutput/NullOutputWrapper.php

```diff
@@ -0,0 +1,23 @@
+<?php declare(strict_types=1);
+
+/* Copyright (c) 2022 - Daniel Weise <daniel.weise@concepts-and-training.de> - Extended GPL, see LICENSE */
+
+namespace CaT\Doil\Lib\ConsoleOutput;
+
+use Symfony\Component\Console\Output\NullOutput;
+
+class NullOutputWrapper extends NullOutput
```

Comment:
What's the purpose of this?

Reply:
The standard NullOutput class has no Section Method in its Interface so i have to mock this to use it in Import.